hadoop-hdds/docs/content/concept/OzonePorts.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Network Ports used by Apache Ozone"
+date: "2025-04-03"
+weight: 9
+menu: 
+  main:
+     parent: Architecture
+summary: Understanding and correctly configuring the network ports used by Apache Ozone is essential for the successful deployment, operation, and maintenance of Apache Ozone clusters.
+
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+This document provides a comprehensive overview of the network ports utilized by Apache Ozone. Due to its distributed nature and the requirement for high performance in handling data-intensive tasks, understanding and correctly configuring these network ports is essential for the successful deployment, operation, and maintenance of Apache Ozone clusters.
+
+# **Ozone Manager (OM)**
+
+| Default Port Number | Configuration Key | Endpoint Protocol | Purpose |
+| :---- | :---- | :---- | :---- |
+| 9862 | ozone.om.address | Hadoop RPC | Primary RPC endpoint for Ozone clients. |
+| 9874 | ozone.om.http-address | HTTP | Web UI for monitoring OM status and metadata. |
+| 9875 | ozone.om.https-address | HTTPS | Secure Web UI for monitoring OM status and metadata. |
```

Comment:
These confs have suffixes: `ozone.om.address.<suffixes>`; see https://github.com/apache/ozone/blob/f6a4a48b0bafe07975ca7ccabbf90a9e1c63b133/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java#L129

Reply:
Thanks for the review! I updated, along with the description in the ozone-default.xml . please check if it's not accurate.